src/FileEncrypter/GpgFileEncrypter.php

```diff
@@ -21,7 +21,7 @@ class GpgFileEncrypter extends AbstractFileEncrypter
     protected function encryptFile($inputFile, $outputFile)
     {
         Process::exec(
-            'gpg2 -v --encrypt {sign} --recipient {recipient} --batch --yes {passphrase} --output {output_file} {input_file}',
+            'gpg2 -v --encrypt {sign} --recipient {recipient} --batch --pinentry-mode loopback --yes {passphrase} --output {output_file} {input_file}',
```

Comment:
I don't think this will work as we don't write the passphrase to a file in the ansible role any more

Reply:
How about using stdin with Symfony process?